test/sagemaker_tests/pytorch/inference/conftest.py

```diff
@@ -24,6 +24,7 @@
 from sagemaker import LocalSession, Session
 from sagemaker.pytorch import PyTorch
 
+from .utils import get_ecr_registry
```

Comment:
can you consolidate this line and the line below?

Reply:
I separated these because the import of `image_utils` imports an entire module.
The import of `get_ecr_registry` only imports a single function.